scripts/gen_airbyte_classes.py

```diff
@@ -480,7 +481,7 @@ def airbyte_repo_path(airbyte_repo_root: Optional[str], tag: str):
         subprocess.call(["git", "clone", "--depth", "1", "--branch", "master", AIRBYTE_REPO_URL])
         os.chdir("./airbyte")
         subprocess.call(["git", "fetch", "--all", "--tags"])
-        subprocess.call(["git", "checkout", "-b" f"tags/{tag}", f"tags/{tag}"])
+        subprocess.call(["git", "checkout", f"-btags/{tag}", f"tags/{tag}"])
```

Comment:
hmm clarity wise i think `"-b", ...` makes more sense but i understand not changing behavior in this PR

maybe leave a comment or ask blame rev author quick